src/resources/action-map/score.js

```diff
@@ -1,86 +1,110 @@
 const assert = require('assert');
 
+const remap = `
+double remap(def value, def map) {
+  for (int i = map.length - 2; i >= 0; i -= 2) {
+    if (map[i] <= value) {
+      return map[i + 1];
+    }
+  }
+  return value;
+}
+`;
+const scoreMapper = (map = []) => {
```

Comment:
```suggestion
const scoreMapper = (map) => {
```
Or when is this required?